### DISCUSSION TITLE
Update Server nok

### DISCUSSION THREAD

**Discussion by valbuz** [codeproject/CodeProject.AI-Server#152] [General]
hi
How is it possible to update the server?
when i click on the version upper right, it says server not reachead!

**Comment by Sailing74** [comment]
I guess that's because the old version is still trying to reach the old site. It's the same for me here.

In the past I've updated to newer versions by installing them on top. I don't know if there is a better way. I can also see that newer versions require newer versions of .NET. So maybe there is even no alternative but to install the newer version on top of the existing one?

Install code for Linux can be found here: https://codeproject.github.io/codeproject.ai/install/install_on_linux.html

Would be great if someone could confirm that this is the preferred/only/best way for updating existing installs.

**Reply by ChrisMaunder** [reply]
https://codeproject.github.io

**Comment by valbuz** [comment]
i have install a completely new installation of a proxmox lxc !

**Reply by ChrisMaunder** [reply]
and all is good?

**Reply by valbuz** [reply]
yes, it's working version: 2.9.2
